build-system/tasks/integration.js

```diff
@@ -68,6 +68,9 @@ integration.flags = {
   'config':
     '  Sets the runtime\'s AMP_CONFIG to one of "prod" (default) or "canary"',
   'coverage': '  Run tests in code coverage mode',
+  'allow_debugger':
+    'Allow "debugger" statements by auto opening devtools. NOTE: This only ' +
```

Comment:
For correct indentation in `gulp help`, add two spaces before the flag description.

Reply:
done.